tests/phpunit/tests/fonts/font-library/wpFontsDir.php

```diff
@@ -69,4 +81,57 @@ function set_new_values( $defaults ) {
 
 		$this->assertSame( static::$dir_defaults, $font_dir, 'The wp_get_font_dir() method should return the default values.' );
 	}
+
+	public function test_should_created_fonts_dir_in_uploads_when_fails_in_wp_content() {
+		// Set the expected results.
+		$upload_dir = wp_upload_dir();
+		$path       = path_join( $upload_dir['basedir'], 'fonts' );
+		$expected   = array(
+			'path'    => $path,
+			'url'     => $upload_dir['baseurl'] . '/fonts',
+			'subdir'  => '',
+			'basedir' => $path,
+			'baseurl' => $upload_dir['baseurl'] . '/fonts',
+			'error'   => false,
+		);
+
+		$this->fake_no_new_directories_in_wp_content();
+		$this->assertFileExists( self::$fake_fonts_file );
+
+		$font_dir = wp_get_font_dir();
+
+		$this->assertDirectoryDoesNotExist( path_join( WP_CONTENT_DIR, 'fonts' ), 'The `wp-content/fonts` directory should not exist' );
+		$this->assertDirectoryExists( $font_dir['path'], 'The `uploads/fonts` directory should exist' );
```

Comment:
```suggestion
		$this->assertDirectoryDoesNotExist( path_join( WP_CONTENT_DIR, 'fonts' ), 'The `wp-content/fonts` directory should not exist.' );
		$this->assertDirectoryExists( $font_dir['path'], 'The `uploads/fonts` directory should exist.' );
```
Adds periods to `$message` for consistency.